lib/fluent/plugin/in_tail.rb

```diff
@@ -35,6 +35,8 @@ class TailInput < Fluent::Plugin::Input
 
     helpers :timer, :event_loop, :parser, :compat_parameters
 
+    RESERVED_CHARS = ['/', '*', '%'].freeze
```

Comment:
These characters should be reserved characters for Unix-like systems.
MSDN says that the following characters are reserved characters in NTFS:

* `< (less than)`
* `> (greater than)`
* `: (colon)`
* `" (double quote)`
* `/ (forward slash)`
* `\ (backslash)`
* `| (vertical bar or pipe)`
* `? (question mark)`
* `* (asterisk)`


Should we also consider them here?

ref: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions

Reply:
`RESERVED_CHARS` is fluentd's reserved chars, not UNIX-like system's or MSDN's one. 
I think it's good that users can use as many chars as possible on their own responsibility. 
WDYT?

Reply:
I'm thinking of these reserved characters handling....
At least, I want to handle `\ (backslash)` as reserved character on Windows, if we can.

Reply:
> `RESERVED_CHARS` is fluentd's reserved chars, not UNIX-like system's or MSDN's one.
> I think it's good that users can use as many chars as possible on their own responsibility.
> WDYT?

 I see. It's reasonable for me. :+1: